apps/block_scout_web/lib/block_scout_web/schema.ex

```diff
@@ -118,6 +118,12 @@ defmodule BlockScoutWeb.Schema do
       resolve(&CeloValidatorGroup.get_by/3)
     end
 
+    @desc "Gets all elected validator signers."
+    field :celo_elected_validators, list_of(:address) do
```

Comment:
Does it return a list of addresses here?

Reply:
It returns the addresses for signers, and there's some associated data. In `types.ex` you can see what data is returned with the address. I added `online` field there. Perhaps there is some way to have inheritance for these types, I should check.